docs/install/install.rst

```diff
@@ -0,0 +1,237 @@
+  use a "K&R C" compiler, or a C++ compiler. The C++ compiler must support C++ 14.
+
+  Development of Freeciv21 is primarily done with :file:`gcc`, the GNU project's excellent C and C++
+  compiler. Microsoft Windows MS Visual C support is under development.
+
+* The :file:`cmake` program.
+
+  Freeciv21 developers generally use :file:`cmake`, the Kitware make program. You can check if you have
+  :file:`cmake` installed on your system by typing the following command. The output should include "Kitware
+  cmake" somewhere and the version should be >=3.12.
+
+.. code-block:: rst
+
+  $ cmake --version
+
+
+* The Ninja cmake build program
+
+
+* :file:`libtool`
+
+  version 2.2 or better
+
+* :file:`libsqlite3`
+
+  http://www.sqlite.org/
+
+  The git version has additional requirements. A release version contains the generated files.
+
+* The programs from GNU :file:`gettext` version 0.15 or better
+
+  Especially the :file:`xgettext` program is required to create the :literal:`*.gmo` files which aren't
+  included in the git tree.
+
+* Lua version 5.3 or higher
+
+  Exact 5.3 is preferred.
+
+* KF 5 Archive Library
+
+  KArchive provides classes for easy reading, creation and manipulation of "archive" formats like ZIP and TAR.
+  
+* SDL2_Mixer
+
+  SDL_mixer is a sample multi-channel audio mixer library.
+
+* Python 3
+
+
+Prerequisites for the client
+****************************
+
+The Freeciv21 project maintains a single Qt client.
+
+* C++ compiler.
+
+  The client is written in C++, so you need an appropriate compiler. In Freeciv21 development, :file:`g++`
+  has been used.
+
+* :file:`Qt5Core`, :file:`Qt5Gui`, and :file:`Qt5Widgets` libraries and headers.
+
+  At least version 5.11 is required.
+
+
+Obtaining the Source Code
+*************************
+
+In order to compile Freeciv21, you need a local copy of the source code. You can download a saved version of 
+the code from the project releases page at https://github.com/longturn/freeciv21/releases. Alternately you 
+can get the latest from the master branch with the :file:`git` program with this command:
+
+.. code-block:: rst
+
+  $ git clone https://github.com/longturn/freeciv21.git
+
+
+.. _configuring:
+
+Configuring
+***********
+
+Configuring Freeciv21 for compilation requires the use of the :file:`cmake` program. To build with defaults enter the following commmand from the freeciv21 directory:
+
+.. code-block:: rst
+
+  $ cmake . -B build -G Ninja
+
+
+To customize the compile, :file:`cmake` requires the use of command line parameters. :file:`cmake` calls 
+them directives and they start with :literal:`-D`. The defaults are marked with :strong:`bold` text.
+
+============================================ =================
+Directive                                    Description
+============================================ =================
+DFREECIV_ENABLE_TOOLS={:strong:`ON`/OFF}     Enables all the tools with one parameter (Ruledit, FCMP, 

```

Comment:
Either you include the `-D` or not, but half of it is weird. Maybe also turn it into a `fixed width` block?